src/cpp/core/include/core/r_util/RSourceIndex.hpp

```diff
@@ -85,14 +85,14 @@ class RSourceItem
 
    RSourceItem(int type,
                const std::string& name,
-               const std::vector<RS4MethodParam>& signature,
```

Comment:
Thanks for changing this, it never felt right having something S4-specific threaded through these functions.